### PR TITLE
using nbsphinx for notebooks

### PR DESCRIPTION
**Pull request by cehbrecht**
This PR adds initial notebooks for documentation (issue #752).

Changes:
* splitted sphinx docs: installation, usage, ...
* added notebook examples: wms (taken from geopython workshop)
* using nbsphinx to render notebooks in sphinx ... notebooks will be run.
* add badges for nbviewer

These notebooks should replace the current "usage" document when complete.

### CODE REVIEW

**Review by tomkralidis — APPROVED** [review]

File: docs/en/development.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+   $ python setup.py test
+
+Or ...
+
+.. code-block:: bash
+
+    # install requirements
+    $ pip install -r requirements.txt
+    $ pip install -r requirements-dev.txt # needed for tests only
```
Comment (fmigneault) [inline docs/en/development.rst:23]: remove `$` to be consistent with other commands below
(more convenient also to copy-paste code when they are not there)

File: docs/en/installation.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+OWSLib requires a Python interpreter, as well as `ElementTree <https://docs.python.org/2/library/xml.etree.elementtree.html>`_ or `lxml <http://lxml.de>`_ for XML parsing.
+
+Install
+-------
+
+PyPI:
+
+.. code-block:: bash
+
+  $ easy_install OWSLib
```
Comment (fmigneault) [inline docs/en/installation.rst:16]: idem

File: docs/en/license.rst [hunk 3]
```diff
@@ -0,0 +1,29 @@
+License
```
Comment (fmigneault) [inline docs/en/license.rst:1]: Use `.. include:: ../../LICENSE` to avoid duplicate

File: docs/en/usage.rst [hunk 4]
```diff
@@ -0,0 +1,564 @@
...
+=====
+
+WMS
+---
+
+Find out what a WMS has to offer. Service metadata:
+
+.. code-block:: python
+
+  >>> from owslib.wms import WebMapService
```
Comment (fmigneault) [inline docs/en/usage.rst:11]: same as for `$`, without the `>>>` is easier to copy-paste by users

**Review by fmigneault — COMMENTED** [review]
only minor formatting changes that would improve docs in my opinion
otherwise looks good

File: docs/en/license.rst (hunk 3 above)
Comment (cehbrecht) [reply docs/en/license.rst:1]: good catch 👍  done.

File: docs/en/development.rst (hunk 1 above)
Comment (cehbrecht) [reply docs/en/development.rst]: done

File: docs/en/installation.rst (hunk 2 above)
Comment (cehbrecht) [reply docs/en/installation.rst]: done

File: docs/en/usage.rst (hunk 4 above)
Comment (cehbrecht) [reply docs/en/usage.rst:11]: skipping `usage.rst` ... it will be replaced with future PRs.

**Comment by cehbrecht** [comment]
test failures not related: `test_ogcapi_features_pygeoapi` needs to be updated.

**Comment by tomkralidis** [comment]
Fixed in master.

**Pull request merged**